content/en/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -0,0 +1,117 @@
+---
+reviewers:
+- dims
+- 44past4
+title: System Logs
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+
+System component logs record things happening in cluster, which can be very useful for debugging.
+You can configure log verbosity to see more or less detail.
+Logs can be as coarse-grained as showing errors within a component, or as fine-grained as showing step-by-step traces of events (like HTTP access logs, pod state changes, controller actions, or scheduler decisions).
+
+<!-- body -->
+
+## Klog
+
+Kubernetes components use the [klog](https://github.com/kubernetes/klog) logging library.
+[klog](https://github.com/kubernetes/klog) configures logging via command line flags exposed by components.
```

Comment:
nit: style: [klog](https://github.com/kubernetes/klog) configures logging by setting command line flags exposed by components.

Reply:
wording: users sets commend line flags, only defines them.

Reply:
Hi @serathius . Here are a few questions and ideas for the page:
I think this paragraph should introduce klog and answer the question, "What is klog"?
The next section could describe how to configure a system component to emit logging.
Can a cluster admin plug in another logging library?
In the configuration section, provide an example of how to set the command line flag (or flags).
Such as:

```shell
kube-apiserver --enable-admission-plugins=NamespaceLifecycle,LimitRanger ...
```

**Some possible additions/edits:**
```
klog is the Kubernetes logging library. [klog](https://github.com/kubernetes/klog)
generates log messages for the Kubernetes system components. [you could identify the components here?]

You can configure the klog library through command line flags.
To enable klog, set the `whatsit-flag` on the system component.
For more information about the `whatsit` flag, see the [Command line tool reference](/docs/reference/command-line-tools-reference/).
```

Reply:
I'm pretty sure that klog is used by all k8s components and is enabled by default. (no klog no logs)

Reply:
Fixed, PTAL